crates/codegen/parser/runtime/src/parse_output.rs

```diff
@@ -21,6 +21,6 @@ impl ParseOutput {
 
     /// Creates a cursor that starts at the root of the parse tree.
     pub fn create_tree_cursor(&self) -> Cursor {
-        return self.parse_tree.create_cursor(Default::default());
+        return self.parse_tree.cursor_with_offset(TextIndex::ZERO);
```

Comment:
❤️ the new `TextIndex::ZERO`! Should we keep its `#[derive(Default)]` in this case? is there a use case for it?

Reply:
There's not reason to not keep it - it allows data types containing this to derive it as well or allows to use it without importing the `TextIndex` directly, like we did before `Default::default()` (it's not important and it's slightly less obvious, but still).

Reply:
Makes sense.